### ISSUE TITLE
File Manager "Connection refused"

### ISSUE THREAD

**Issue by marcus89** [moddevices/mod-ui#147]
I want to upload Midi file for the carla-midifile plugin. For this, I need to upload file from the UI on "File Manager" page.
However, The "File Manager" page does not load.

Using wireshark, I can see that "File Manager" page try to connect to port 8081 (from html/js/file_manager.js file). In addition, when starting mod-ui, I can not see any application listening on port 8081.

Is this something normal?

Thanks for you help.

Marius

**Comment by falkTX** [comment]
yes, that file manager is just https://github.com/moddevices/browsepy on that port, it needs to be running separately

**Comment by marcus89** [comment]
Yow, thanks for the fast answer.

Understood. I made it run and now, I can see it on the File Manager page. However, the carla-midifile plugin does not see any of my midi file. Do I need to put them on a specific folder?
